prolog/tokenize_numbers.pl

```diff
@@ -0,0 +1,7 @@
+:- module(tokenize_numbers, [number//1]).
```

Comment:
I don't get why this is its own module...

How does this get included in the main module?  I don't see an import there.  Maybe I'm missing some Prolog magic?

Reply:
Sorry, my git fu failed, I thought I removed that file

Reply:
fixed